templates/apps/create_base.html

```diff
@@ -122,6 +122,15 @@ <h1 class="h3 mb-3 card-title">Create {{ form.model_name }}</h1>
         }
     }
 
+    // SS-1176 Users are able to set a custom default start URL for their apps
+    const subdomainInput = document.getElementById("id_subdomain");
+    const customDefaultUrlFormModifiedUrlText = document.getElementById("id_custom_default_url_prepend");
+
+    function updateUrlWithSubdomain() {
+        const subdomainValue = subdomainInput.value.trim();
+        const displayValue = subdomainValue || "subdomain_name";
+        customDefaultUrlFormModifiedUrlText.innerHTML = `${displayValue}.{% get_setting 'DOMAIN' %}/</b>`;
```

Comment:
```suggestion
        try {
            const subdomainValue = subdomainInput.value.trim();
            const displayValue = subdomainValue || "subdomain_name";
            customDefaultUrlFormModifiedUrlText.innerHTML = `${displayValue}.{% get_setting 'DOMAIN' %}/</b>`;
        } catch (error) {
            console.log("No custom url subpath option in this app type.");
        }
```

Reply:
Thanks a lot. With it, it passed all the e2e test cases.